<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-04T03:12:48Z" name="Suite" finished-at="2024-12-04T03:13:30Z" duration-ms="42360">
    <groups>
    </groups>
    <test started-at="2024-12-04T03:12:48Z" name="Run on Chrome" finished-at="2024-12-04T03:13:30Z" duration-ms="42360">
      <class name="com.nopcommerce.order.Order_01_Order">
        <test-method is-config="true" signature="beforeClass(java.lang.String)[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:12:48Z" name="beforeClass" finished-at="2024-12-04T03:13:16Z" duration-ms="28245" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="Order_01_Add_To_Cart()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:16Z" name="Order_01_Add_To_Cart" finished-at="2024-12-04T03:13:30Z" duration-ms="14101" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.nopcommerce.order.Order_01_Order.Order_01_Add_To_Cart(Order_01_Order.java:69)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Order_01_Add_To_Cart -->
        <test-method signature="Order_02_Edit_Production_In_Shopping_Cart()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:30Z" name="Order_02_Edit_Production_In_Shopping_Cart" finished-at="2024-12-04T03:13:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Order_02_Edit_Production_In_Shopping_Cart -->
        <test-method signature="Order_03_Remove_From_Cart()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:30Z" name="Order_03_Remove_From_Cart" finished-at="2024-12-04T03:13:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Order_03_Remove_From_Cart -->
        <test-method signature="Order_04_Update_Shopping_Cart()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:30Z" name="Order_04_Update_Shopping_Cart" finished-at="2024-12-04T03:13:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Order_04_Update_Shopping_Cart -->
        <test-method signature="Order_05_Checkout()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:30Z" name="Order_05_Checkout" finished-at="2024-12-04T03:13:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Order_05_Checkout -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.nopcommerce.order.Order_01_Order@124c278f]" started-at="2024-12-04T03:13:30Z" name="afterClass" finished-at="2024-12-04T03:13:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.nopcommerce.order.Order_01_Order -->
    </test> <!-- Run on Chrome -->
  </suite> <!-- Suite -->
</testng-results>
